dataflow/manual/examples/BusyExprSimple2.java

```diff
@@ -0,0 +1,13 @@
+class Test {
```

Comment:
Should a reference to this example be in the manual?

Reply:
This is used for BusyExprplayground.java. There's already one more complicated test file in manual.

Reply:
The playground shouldn't use any file, just like in https://github.com/eisop/checker-framework/blob/master/dataflow/src/main/java/org/checkerframework/dataflow/cfg/playground/LiveVariablePlayground.java#L21
Also, at the moment the Playground is using `BusyExprSimple.java` and not `BusyExprSimple2.java`. If you want another test file with the playground, why not put it there, instead of in the manual?

Reply:
Oh! I will delete it. Adding two examples may make the manual there a bit crowded.